src/components/custom-tom-select.scss

```diff
@@ -145,7 +145,7 @@
   align-self: stretch;
 }
 .ts-wrapper.plugin-remove_button .item .remove:hover {
-  background-color: var(--rms-colors-neutral-70);
+  background-color: var(--rm-color-neutral-70);
 }
 .ts-wrapper.plugin-remove_button .item.active .remove {
   border-left-color: #cacaca;
```

Comment:
@Jeremy-Walton Can we use our tokens here instead of the static styles for color.